gapis/api/vulkan/api/draw_commands.api

```diff
@@ -147,6 +147,9 @@ cmd void vkCmdBindVertexBuffers(
 
 
 sub void dovkCmdDraw(ref!vkCmdDrawArgs draw) {
+  readDrawState()
```

Comment:
These are likely going to have a noticeable performance hit on trace time.
Can we mark them as @spy_disabled.